client/lib/user/user.js

```diff
@@ -5,14 +5,16 @@ var store = require( 'store' ),
 	debug = require( 'debug' )( 'calypso:user' ),
 	config = require( 'config' ),
 	qs = require( 'qs' ),
-	isEqual = require( 'lodash/lang/isEqual' );
+	isEqual = require( 'lodash/lang/isEqual' ),
+	localforage = require( 'localforage' );
 
 /**
  * Internal dependencies
  */
 var wpcom = require( 'lib/wp' ),
 	Emitter = require( 'lib/mixins/emitter' ),
-	userUtils = require( './shared-utils' );
+	userUtils = require( './shared-utils' ),
+	localforageConfig = require( 'state/initial-state' );
```

Comment:
Shouldn't this be `localforageConfig = require( 'state/initial-state' ).localforageConfig` ?